Add component City to storybook.

6kn4eakfr4s opened the pull request:
Ref #6

kaidev1024 commented on src/common/City.stories.tsx:
@6kn4eakfr4s for size, just `md` and `lg` is enough. can you add test for default color and set the color to blue? Add a screenshot to the description.

6kn4eakfr4s replied:
I added an argument for color and set it to be blue for default. I also added a interaction in storybook to test the color.
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/96273cc3-9ac0-4169-b138-ffb317eac036" />

kaidev1024 commented on src/common/City.stories.tsx:
```diff
@@ -11,29 +12,23 @@ export default meta;
+    
+    // Assert that the fill attribute is set to blue
+    expect(svgPath).not.toBeNull();
+    expect(svgPath).toHaveAttribute('fill', 'blue');
```
@6kn4eakfr4s color should be set in className with tailwind. `className = 'fill-blue-500'`

kaidev1024 commented on src/util.ts:
```diff
@@ -2,6 +2,7 @@
 export interface SvgProps {
   className?: string;
   size?: number;
+  color?: string;
```
@6kn4eakfr4s no need to add color in props. define color in className with tailwind

6kn4eakfr4s commented:
@kaidev1024 I installed tailwindcss and changed the default colors to blue. I also removed unnecessary sizes in storybook. Could you please take a look?
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/1ecd9c9f-d359-4f3d-962e-6f6eb16f5709" />

6kn4eakfr4s commented:
@kaidev1024 I added the UI test for color.
<img width="1726" alt="image" src="https://github.com/user-attachments/assets/69be8302-414a-4226-87ec-ac71d631ddcc" />

kaidev1024 commented on src/common/City.tsx:
```diff
@@ -1,7 +1,8 @@
 
 function City({
-  className = '',
+  className = "fill-blue-500",
```
@6kn4eakfr4s unit testing is not suppose to change the original components. You can pass blue color to the City and render it on storybook.
Render two colors on storybook, blue and default.

6kn4eakfr4s replied:
@kaidev1024 I have changed the default color back to black and added blue as separate stories. Could you please take another look?
